frontend/src/components/PrefillTemplateColumns.tsx

```diff
@@ -46,12 +47,13 @@ interface SelectInstrumentTypeProps extends SelectProps {
 }
 function SelectInstrumentType({ type, ...props }: SelectInstrumentTypeProps) {
     const listInstrumentTypesCallback = useCallback(() => store.dispatch(api.instrumentTypes.list({ platform__name: type })), [type])
-    
+
     const [instrumentTypes, setInstrumentTypes] = useState<InstrumentType[]>([])
 
     useEffect(() => {
         listInstrumentTypesCallback().then((response) => {
-            setInstrumentTypes(response.data.results)
+            let results = response.data.results.filter((instrument: FMSInstrumentType) => instrument.is_in_prefiller)
```

Comment:
Add filtering code here. You need to check on the step make sure this is the sample QC step... then get only the right stuff though.